docs/docs/migration/migration-from-v6-to-v7.md

```diff
@@ -136,38 +215,735 @@ It automatically installs the new versions of all `@comet` libraries, runs an ES
+
+Check if the styling still looks as intended in your application.
+
+### @comet/admin-color-picker
+
+#### Prop renames and removals
+
+<details>
+
+<summary>Expand for details</summary>
+
+-   `ColorPicker`: Replace `componentsProps` with `slotProps`
+-   `ColorPicker`: Remove the `clearable` prop. The clear button will be shown automatically for optional fields
+
+</details>
+
+### @comet/admin-date-time
+
+#### Change the value type
+
+The value returned by `DatePicker` and `DateRangePicker` is now a `string` (previously it was a `Date`).
+
+The code that handles values from these components needs to be adjusted.
+**This may include how the values are stored in or sent to the database.**
+
+**Required Admin Changes:**
+
+```diff
+-   const [date, setDate] = useState<Date | undefined>(new Date("2024-03-10"));
++   const [date, setDate] = useState<string | undefined>("2024-03-10");
+    return <DatePicker value={date} onChange={setDate} />;
+```
+
+```diff
+    const [dateRange, setDateRange] = useState<DateRange | undefined>({
+-       start: new Date("2024-03-10"),
+-       end: new Date("2024-03-16"),
++       start: "2024-03-10",
++       end: "2024-03-16",
+    });
+    return <DateRangePicker value={dateRange} onChange={setDateRange} />;
+```
+
+#### Prop renames and removals
+
+<details>
+
+<summary>Expand for details</summary>
+
+-   `DatePicker`:
+
+    -   Replace the `componentsProps` prop with `slotProps`
+    -   Remove the `DatePickerComponentsProps` type
+    -   Remove the `clearable` prop. The clear button will be shown automatically for all optional fields.
+
+-   `DateRangePicker`:
+
+    -   Replace the `componentsProps` prop with `slotProps`
+    -   Remove the `DateRangePickerComponentsProps` type
+    -   Rename the `calendar` class-key to `dateRange`
+    -   Remove the `clearable` prop. The clear button will be shown automatically for all optional fields.
+
+-   `DateTimePicker`:
+
+    -   Replace the `componentsProps` prop with `slotProps`
+    -   Remove the `DateTimePickerComponentsProps` type
+    -   Replace the `formControl` class-key with two separate class-keys: `dateFormControl` and `timeFormControl`
+    -   Remove the `clearable` prop. The clear button will be shown automatically for all optional fields.
+
+-   `TimePicker`:
+
+    -   Remove the `clearable` prop. The clear button will be shown automatically for all optional fields.
+
+-   `TimeRangePicker`:
+
+    -   Replace the `componentsProps` prop with `slotProps`
+    -   Remove the `TimeRangePickerComponentsProps` and `TimeRangePickerIndividualPickerProps` types
+    -   Replace the `formControl` class-key with two separate class-keys: `startFormControl` and `endFormControl`
+    -   Replace the `timePicker` class-key with two separate class-keys: `startTimePicker` and `endTimePicker`
+    -   Remove the `clearable` prop. The clear button will be shown automatically for all optional fields.
+
+</details>
+
+## Site
```

Comment:
Site is still unfinished. I think we need more info about the switch to App Router. Also currently we only support App Router but (hopefully) in the next beta version we will also support Pages Router again